wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -205,6 +209,30 @@ instance FromHttpApiData (V1 Core.Address) where
 instance ToHttpApiData (V1 Core.Address) where
     toQueryParam (V1 a) = sformat build a
 
+-- | Represents according to 'apiTimeFormat' format.
+instance ToJSON (V1 Core.Timestamp) where
+    toJSON timestamp =
+        let utcTime = timestamp ^. _V1 . Core.timestampToUTCTimeL
+        in  String $ showApiUtcTime utcTime
+
+-- | Parses from both UTC time in 'apiTimeFormat' format
+-- and POSIX (fractional) timestamp in seconds.
+instance FromJSON (V1 Core.Timestamp) where
+    parseJSON v = V1 <$> asum
+        [ utcTimeParser v
```

Comment:
Silly question: if our `ToJSON` formats using `UTCTime`, what are we gaining in supporting the `timePosixParser` here?

Reply:
Um right, it's actually useless